src/app/scripts/cac/pages/cac-pages-map.js

```diff
@@ -154,8 +154,26 @@ CAC.Pages.Map = (function ($, Handlebars, _, moment, MapControl, Routing, MockDe
         var $container = $('<div></div>').addClass('destinations');
         $.each(destinations, function (i, destination) {
             var $destination = $(CAC.Map.Templates.destinationBlock(destination));
-            $destination.click(function (event) {
-                // TODO: What to do on click?!?!
+
+            $destination.click(function () {
```

Comment:
The unused `event` was throwing a linter warning; I imagine that's why it was removed here (I've already removed it in #85).